### DISCUSSION TITLE
Criar Blog para Devlogs

### DISCUSSION THREAD

**Discussion by CodyKoInABox** [Equipe-CyberPAC/discussions#2] [Ideas]
Criar um blog usando Astro para escrever devlogs

**Comment by CodyKoInABox** [comment]
https://docs.astro.build/en/tutorial/0-introduction/

**Reply by ruan-cardozo** [reply]
https://docs.astro.build/pt-br/core-concepts/framework-components/

**Comment by CodyKoInABox** [comment]
Aqui tem um exemplo de mais ou menos como seria o nosso: 
https://withastroastroypszdu-zzum--3000--d5d65229.local-credentialless.webcontainer.io
(usando o framework Astro)

**Reply by CodyKoInABox** [reply]
esquece o link nao funciona

**Reply by ruan-cardozo** [reply]
https://github.com/KumailP/kumails-blog

**Comment by ruan-cardozo** [comment]
![image](https://github.com/Equipe-CyberPAC/discussions/assets/110867639/9292e1d0-a931-486f-a883-c70b3f87260f)
Fiz alguns estudos consegui rodar local esse blog com astro, acho que da pra nos trabalhar nele como base e ir acrescentando coisas, ou se quiser podemos fazer do zero também

**Reply by ruan-cardozo** [reply]
E basicamente só a configuração e toda a parte burocrática de configurar as coisas, agora e talvez trabalhar na estilização e tudo mais, preciso se aprofundar um pouco mais... se quiser fazer uma call durante essa semana pra discutir algumas coisas que descobriu

**Comment by ruan-cardozo** [comment]
https://www.markdownguide.org/cheat-sheet/ dica de site para os markodown para as postagens no blog
